### PR TITLE
Blog: Reporting Competition

### PR DESCRIPTION
**Pull request by psiinon**
Planned for 12th but can move forward or back depending on the feedback received :)

Can post a screenshot here if it helps, or does everyone pull PRs down and build locally anyway?
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/blog/2021-03-12-report-competition/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## New Reporting Add-on
+In order to be able to generate more modern and flexible reports we have released a new Reporting Add-on at [ZAPCon 2021](https://zapcon.io): that is now available on the ZAP Marketplace: [Report Generation](https://www.zaproxy.org/docs/desktop/addons/report-generation/).
+
+This add-on is still at an early stage - for a list of enhancements planned see the related [issue tracker](https://github.com/zaproxy/zaproxy/issues/6483).
+
+The key ones related to this competition (and which will be prioritised) are:
+
+#### Support resources like JS and images
+This is key for supporting modern HTML reports which will need charts, custom images and support for 3rd party JavaScript libraries.
```
Comment (kingthorin) [inline site/content/blog/2021-03-12-report-competition/index.md:54]:
```suggestion
This is key for supporting modern HTML reports which will need charts, custom images, and support for 3rd party JavaScript libraries.
```

File: site/content/blog/2021-03-12-report-competition/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+ - simon
+---
+
+## The Competition
+
+![](./images/social.png)
+
+We have a new [Report Generation](https://www.zaproxy.org/docs/desktop/addons/report-generation/) add-on (detailed below) 
+but while it is much more flexible, the current report templates it includes are still very basic. 
+Lets be honest - creating stylish reports is probably not one of the ZAP core team's strengths ;)
```
[the remark this answers is not in the document]
Comment (psiinon) [reply site/content/blog/2021-03-12-report-competition/index.md:23]: Ha! Definitely

**Comment by psiinon** [comment]
Updated based on feedback received :)

**Comment by psiinon** [comment]
Ideally https://github.com/zaproxy/zap-extensions/pull/2758 will be merged before this is published

File: site/content/blog/2021-03-12-report-competition/index.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+* More important: 16.1%
+* Very important: 61.3%
+
+#### Any other reporting features you would like us to include?
+This was a free text field and the answers are included without any changes:
+
+* Be able to translate
+* Branding. Not with the intention of branding for a hired engagement, but for internal reporting. 
+* thx!
+* The support for certain report formats like Hipaa, is27001, owasp Top10, pci, .... would be really cool :-)
```
Comment (kingthorin) [inline site/content/blog/2021-03-12-report-competition/index.md:139]:
```suggestion
* The support for certain report formats like HIPAA, ISO27001, OWASP Top10, PCI, .... would be really cool :-)
```

Comment (kingthorin) [reply site/content/blog/2021-03-12-report-competition/index.md:139]: I guess this is a quote, but I'd suggest at least fixing the "ISO" one.

Comment (psiinon) [reply site/content/blog/2021-03-12-report-competition/index.md:139]: "This was a free text field and the answers are included without any changes" I'd rather not change anything tbh

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2021-03-12-report-competition/index.md (hunk 3 above)
Comment (kingthorin) [reply site/content/blog/2021-03-12-report-competition/index.md:139]: Ok

**Review by kingthorin — APPROVED** [review]

**Pull request merged**